Steps4Impact/Journey/Cells/CurrentMilestoneCell.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  MilestoneCell.swift
+//  Steps4Impact
+//
+//  Created by Aalim Mulji on 11/17/19.
+//  Copyright © 2019 AKDN. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+struct CurrentMilestoneContext : CellContext {
+  var identifier: String = CurrentMilestoneCell.identifier
+  var sequence = 0
+  var distance = 0
+  var name = ""
+  var flagName = ""
+  var journeyMap = ""
+  var description = ""
+  var title = ""
+  var subTitle = ""
+  var media = ""
+  var content = ""
+  var status : MilestoneStatus = .current
+  var progress : CGFloat = 0.0
+}
+
+class CurrentMilestoneCell: ConfigurableTableViewCell {
+
+  static let identifier: String = "CurrentMilestoneCell"
+
+  let circle : UIView = {
+    var view = UIView(frame: .zero)
+    view.translatesAutoresizingMaskIntoConstraints = false
+    view.layer.cornerRadius = 12
+    return view
+  }()
+
+  let verticalBar : UIView = {
+    var view = UIView(frame: .zero)
+    view.translatesAutoresizingMaskIntoConstraints = false
+    view.frame.size.width = 2
+    return view
+  }()
+
+  let verticalProgressBar : UIView = {
+    var view = UIView(frame: .zero)
+    view.translatesAutoresizingMaskIntoConstraints = false
+    view.frame.size.width = 2
+    return view
+  }()
+
+  let progressCircle : UIView = {
+    var view = UIView(frame: .zero)
+    view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
you don't need `translatesAutoresizingMaskIntoConstraints` when using snapkit - it does it for you when adding constraints